client/java/src/main/java/com/linecorp/centraldogma/client/AbstractWatcher.java

```diff
@@ -82,9 +96,16 @@ private enum State {
     @Nullable
     private volatile CompletableFuture<?> currentWatchFuture;
 
+    // unix epoch seconds
+    // In Prometheus, it is common to handle data with second precision,
+    // so we intentionally use second precision.
+    // ref: https://prometheus.io/docs/prometheus/latest/querying/functions/#time
```

Comment:
Optional) I think `TimeGauge` can be used if we want to make this metric implementation-agnostic.

e.g.
```
TimeGauge.builder("...", ..., TimeUnit.MILLISECONDS, ...)
```